### ISSUE TITLE
Feature: Mint ERC721/1155 tokens

### ISSUE THREAD

**Issue by jxom** [paradigmxyz/rivet#40]
Depends on #39 

### The Request

The ability to (quickly) mint more ERC721 or ERC1155 tokens within Rivet.

### Problem

As a Dapp developer, I may be creating a token-gated Dapp, which may require at least one of an ERC721 token. There is currently no way to be able to mint yourself any ERC721 or ERC1155 from any contract address from within Rivet.

### Solution

From within the ERC721/1155 token list in #39, it would be nice to have an "Add or import NFT" button, that navigates to a screen where I can enter the contract address & token id/quantity, so that I can import any arbitrary ERC721/1155 token.

**Comment by mahmudsudo** [comment]
can i take on this ?
